rafiki/admin/admin.py

```diff
@@ -69,28 +69,45 @@ def create_user(self, email, password, user_type):
     ####################################
 
     def create_train_job(self, user_id, app,
-        task, train_dataset_uri, test_dataset_uri, budget):
+        task, train_dataset_uri, test_dataset_uri, models):
         
         # Compute auto-incremented app version
         train_jobs = self._db.get_train_jobs_of_app(app)
         app_version = max([x.app_version for x in train_jobs], default=0) + 1
 
-        # Ensure that there are models associated with task
-        models = self._db.get_models_of_task(task)
+        # Ensure that models are specified
```

Comment:
Should `model` option during train job creation be optional? If app developers don't specify a list of models to train, should Rafiki help them choose?

@nudles

Reply:
yes. it should be optional.
the ultimate goal is to let rafiki decide the model(s) for app developers who are not ml experts.

Reply:
Fixed this such that models is optional (ie models=None)